apps/price_pusher/price-config.stable.sample.yaml

```
@@ -1,19 +1,66 @@
+    time_difference: 5
+    price_deviation: 0.03
+- alias: GME/USD
+  id: 6f9cd89ef1b7fd39f667101a91ad578b6c6ace4579d5f7f285a4b06aa4504be6
+  time_difference: 10     # pushes if on-chain is >10s behind
+  price_deviation: 0.05   # 0.05% deviation trips a push
+  confidence_ratio: 200   # effectively ignore this trigger
+  early_update:
+    time_difference: 5
+    price_deviation: 0.03
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (2 > 0)

(empty-lines)